docs/DeveloperGuide.adoc

```diff
@@ -315,33 +324,333 @@ _{More to be added}_
 
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
+
 [discrete]
-=== Use case: Delete person
+=== Use case: Add a team member
 
-*MSS*
+*Main Success Scenario*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to add a team member
+2. User specifies the name of the team member
 +
-Use case ends.
+Use case ends
 
 *Extensions*
 
-[none]
-* 2a. The list is empty.
+2. The user does not specify name
+3. +work shows an error message
```

Comment:
Numbering format is wrong. Should start from the step number where the extension occurs. Refer to [this](https://nus-cs2103-ay1920s1.github.io/website/schedule/week7/topics.html).